### ISSUE TITLE
Container + event ignores threshold

### ISSUE THREAD

**Issue by Vielmond** [tuupola/jquery_lazyload#247]
Lazyload seems to be misbehaving when you trigger it through an event. In my case I've set up a horizontal slider. The parent container (which I'm referring to in the plugin settings) has the viewport width. The child container contains the images I'm loading, it has the images added width and slides around upon clicking on the slider navigation buttons, keyboard arrows, etc.

The initial load works flawlessly and the custom threshold setting is respected, after that—when the plugin gets fired by the sliding event—the threshold gets ignored, causing only the images in the viewport to be loaded.

**Comment by tuupola** [comment]
URL of the page you have problem with? The described problem should not be logically possible. For the plugin there is no difference in the event whether is was triggered by browser or triggered manually. Maybe it is somehow caused by the container.

**Comment by Vielmond** [comment]
I'm not entirely sure of how I managed to do this either. I suspect it's making this usage of the plugin added to a conflict with some other piece of javascript.

Demo: http://en.vielmond.com.br/container-demo/

I'm using the velocity plugin, a handler for swyping (hammer.min.js), a piece of code to handle the the slider and another to initialize the lazyload and deal with some UI (main.js).

The threshold is set in 1800, and I've loaded the images there twice just to help the test.

**Comment by Vielmond** [comment]
Quick update: Maybe I misunderstood how events are supposed to work? I assumed I'd get the same reaction from the plugin I get from scrolling, which is load whatever is in the viewport added to threshold and failure_limit.

What I really get when I trigger the event at the images-yet-to-load is a massive load of all of them. Visible or not, anywhere all the other images will load, as seen on this example: http://en.vielmond.com.br/container-demo-2/

I've stripped this one of the other pieces of code, and my trigger is a click on the ugly red button. On the first example I've set the trigger on the .active image to get some measure of control (but then nothing else loads), on the second it's set on the same images from before just like in the docs example.
